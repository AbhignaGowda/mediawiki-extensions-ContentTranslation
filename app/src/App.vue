<template>
  <div id="cxdashboard" class="container">
    <cx-header />
    <div class="row cx-translation-header">
      <mw-button
        progressive
        large
        :icon="mwIconAdd"
        :label="$i18n('cx-create-new-translation')"
        class="col-md-4 col-xs-12 col-lg-3 col-offset-lg-1 mb-4 mt-4"
      >
      </mw-button>
    </div>
  </div>
</template>

<script>
import MwButton from "./lib/mediawiki.ui/components/MWButton";
import CxHeader from "./components/CXHeader";
import { mwIconAdd } from "./lib/mediawiki.ui/components/icons";
import "./lib/mediawiki.ui/grid.less";

export default {
  name: "cx-dashboard",
  components: { MwButton, CxHeader },
  data: () => ({ mwIconAdd })
};
</script>

<style lang="less">
body {
  font-family: sans-serif;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
  color: #54595d;
  background-color: #eaecf0;
  margin: 0;
  padding: 0;
}
</style>
