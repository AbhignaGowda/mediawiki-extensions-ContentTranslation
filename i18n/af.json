{
	"@metadata": {
		"authors": [
			"Slashme",
			"Winstonza",
			"Amire80",
			"Purodha",
			"Fwolff",
			"Oesjaar"
		]
	},
	"cx": "Vertaal bladsy",
	"cx-dashboard-header": "Vertalings",
	"cx-javascript": "Hierdie werktuig werk nie sonder JavaScript nie.",
	"cx-specialpage-enable-betafeature": "[[mw:ContentTranslation|Content Translation]] is 'n program om vinnig bladsye in u taal te vertaal. [$1 Probeer dit gerus nou!]",
	"cx-license-agreement": "Deur op die \"$1\"-knoppie te klik, stem jy toe tot die [//wikimediafoundation.org/wiki/Terms_of_Use Gebruiksvoorwaardes] en jy stem onherroeplik toe dat jou bydraes wat deur die vertalingswerktuig gemaak is, onder die [https://creativecommons.org/licenses/by-sa/3.0/ CC BY-SA 3.0 Lisensie] en die [//en.wikipedia.org/wiki/Wikipedia:Text_of_the_GNU_Free_Documentation_License GNU Vrye Dokumentasie-lisensie] gepubliseer word, met die verstandhouding dat 'n webskakel genoegsaam is vir erkenning in terme van CC BY-SA 3.0.",
	"cx-error-server-connection": "Fout: kon nie 'n verbinding met die bediener bewerkstellig nie.",
	"cx-desc": "Maak dit maklik om inhoudbladsye te [[Special:ContentTranslation|vertaal]].",
	"cx-header-progressbar-text": "$1% {{PLURAL:$1|vertaal}}",
	"cx-header-progressbar-text-mt": "($1% vanaf {{PLURAL:$1|outomatiese vertaling}})",
	"cx-header-translation-center": "Vertalingsentrum",
	"cx-header-new-translation": "Nuwe vertaling",
	"cx-header-all-translations": "Alle vertalings",
	"cx-source-view-page": "Wys bladsy",
	"cx-publish-page-success": "Bladsy is by $1 gepubliseer",
	"cx-publish-page-error": "'n Fout het voorgekom toe die bladsy gestoor is. Probeer gerus om die bladsy weer te publiseer. Foutː $1",
	"cx-publish-button": "Publiseer vertaling",
	"cx-publish-button-publishing": "Besig om te publiseer…",
	"cx-lost-session-draft": "Die sessie het uitgetel. U wysigings kan nie gestoor word nie tensy u weer aanmeld in 'n ander venster.",
	"cx-lost-session-publish": "Die sessie het uitgetel. U kan nie die vertaling publiseer nie tensy u weer aanmeld in 'n ander venster.",
	"cx-publish-summary": "Geskep deur die bladsy \"$1\" te vertaal",
	"cx-publish-gt-no-permission-to-move-description": "Vra asb. 'n ervare gebruiker van hierdie webwerf om jou te help om jou vertaling deur te gaan en dit in die hoofartikelruimte te publiseer.",
	"cx-publish-gt-first-step-title": "Die vertaling is as 'n konsepartikel onder jou gebruikersbladsy gepubliseer sodat hy hersien kan word.",
	"cx-publish-gt-first-step-description": "Klik op die \"$1\" aksie onder hierdie spyskaart om die inhoud vir alle gebruikers beskikbaar te maak as 'n normale bladsy.",
	"cx-publish-gt-move-page-title": "Skuif die bladsy",
	"cx-publish-gt-move-page-description": "Kies die \"{{int:blanknamespace}}\" ruimte en gee jou bladsy 'n gepaste titel.",
	"cx-publish-gt-moved-title": "Die vertaalde bladsy is geskuif",
	"cx-publish-gt-moved-description": "Klik hierdie skakel om jou nuwe vertaalde bladsy te lees en verbeter",
	"cx-publish-gt-published-title": "Veels geluk, die bladsy is gepubliseer",
	"cx-publish-gt-published-description": "Jy kan \"$1\" klik om die bladsy verder te verbeter. Maak seker dat dit natuurlik voorkom.",
	"cx-translation-add-translation": "Voeg vertaling by",
	"cx-translation-target-page-exists": "'n Bladsy met die titel [$1 $2] bestaan reeds in die teikenwiki. Oorweeg dit om 'n ander titel te kies.",
	"tag-contenttranslation": "[[{{MediaWiki:cx-descriptionpagelink}}|Inhoudvertaling]]",
	"tag-contenttranslation-description": "Die inhoud is deur die Inhoudsvertaling werktuig vanuit 'n ander taal vertaal.",
	"cx-source-loading": "Laai $1",
	"cx-beta": "Inhoudsvertaling",
	"cx-beta-desc": "'n Werktuig om artikels vinnig in jou taal te vertaal. Begin by [[Special:MyContributions|jou gebruikersbydraes bladsy]] en wysig hulle met ons sy-aan-sy redigeerwerktuig wat spesifiek ontwerp is vir vertaling. Van die werktuie sal net vir sekere tale beskikbaar wees.",
	"cx-entrypoint-title": "Vertaal hierdie bladsy na $1",
	"cx-entrypoint-dialog-page-doesnt-exist-yet": "Die bladsy bestaan nog nie in $1 nie. Wil jy dit skep?",
	"cx-entrypoint-dialog-title-in": "Titel vir die nuwe bladsy in $1:",
	"cx-entrypoint-dialog-button-create-from-scratch": "Skep van nuuts af",
	"cx-entrypoint-dialog-button-translate-from": "Vertaal uit $1",
	"cx-special-login-error": "Jy moet aangemeld wees om herdie bladsy te vertaal. [$1 Meld aan]",
	"cx-tools-instructions-text1": "Klik paragrawe om te vertaal",
	"cx-tools-instructions-text2": "Jy hoef hulle nie almal by te voeg nie",
	"cx-tools-instructions-text3": "Laat die teks natuurlik lees",
	"cx-tools-instructions-text4": "Outomatiese vertaling is nuttig, maar jy moet nog steeds die teks hersien om dit akkuraat te maak.",
	"cx-tools-instructions-text5": "Publiseer die vertaling",
	"cx-tools-instructions-text6": "As jy gelukkig is met die resultaat, druk \"{{int:cx-publish-button}}\".",
	"cx-tools-searchbox-text": "Soek 'n woord",
	"cx-tools-view-guidelines": "Sien die vertalingsriglyne",
	"cx-tools-dictionary-title": "Definisie",
	"cx-tools-link-title": "Skakel",
	"cx-tools-link-add": "Voeg 'n skakel by",
	"cx-tools-link-remove": "Verwyder skakel",
	"cx-tools-mt-title": "Outomatiese vertaling",
	"cx-tools-mt-use-source": "Gebruik bronteks",
	"cx-tools-mt-reset": "Herstel vertaling",
	"cx-tools-mt-provider-title": "Gebruik $1",
	"cx-tools-mt-dont-use": "Moenie outomatiese vertaling gebruik nie",
	"cx-tools-mt-set-default": "Hou as verstek",
	"cx-tools-categories-count-message": "{{PLURAL:$1|$1 kategorie|$1 kategorieë|0=Geen kategorieë}}",
	"cx-stats-title": "Vertaalstatistieke",
	"cx-tools-reference-title": "Verwysing",
	"cx-tools-reference-add": "Voeg verwysing by",
	"cx-tools-reference-remove": "Verwyder verwysing",
	"cx-tools-link-instruction-shortcut": "Shift + klik enige skakel om oop te maak",
	"cx-tools-link-hover-tooltip": "Klik om oop te maak",
	"cx-warning-unsaved-translation": "Jy het ongestoorde vertalings.",
	"cx-error-page-not-found": "Die \"$1\" bladsy kon nie in $2 Wikipedia gevind word nie",
	"cx-sourceselector-dialog-new-translation": "Nuwe vertaling",
	"cx-sourceselector-dialog-button-start-translation": "Begin vertaling",
	"cx-sourceselector-dialog-button-cancel": "Kanselleer",
	"cx-sourceselector-dialog-source-language-label": "Vanaf:",
	"cx-sourceselector-dialog-target-language-label": "Na:",
	"cx-sourceselector-dialog-source-title-placeholder": "Soek bronbladsy",
	"cx-sourceselector-dialog-target-title-placeholder": "Titel van vertaling",
	"cx-sourceselector-dialog-error-page-and-title-exist": "Hierdie titel bestaan reeds in [$1 $2] vir [$3 'n ander bladsy]",
	"cx-sourceselector-dialog-error-page-exists": "Die artikel [$1 bestaan reeds] in $2",
	"cx-sourceselector-dialog-error-title-in-use": "Die titel vir die nuwe bladsy is [$1 reeds in gebruik]",
	"cx-sourceselector-dialog-error-no-source-article": "Die bladsy wat jy wil vertaal bestaan nog nie in $1 nie",
	"cx-mt-abuse-warning-title": "Die vertaling {{PLURAL:$1|bevat}} $1% ongewysigde outomaties vertaalde teks",
	"cx-mt-abuse-warning-text": "Die outomatiese vertalings word net as 'n beginpunt verskaf. Maak seker dat die inhoud akkuraat is, en natuurlike taal is.",
	"cx-publish-captcha-title": "Sekuriteitsvraag",
	"cx-dashboard-sidebar-title": "Hulp nodig met vertaling?",
	"cx-dashboard-sidebar-information": "Meer inligting oor dié funksionaliteit",
	"cx-dashboard-sidebar-stats": "Statistiek",
	"cx-dashboard-sidebar-feedback": "Gee gerus terugvoer",
	"cx-translation-filter-all-translations": "Alle vertalings",
	"cx-translation-filter-published-translations": "Gepubliseer",
	"cx-translation-filter-draft-translations": "In wording",
	"cx-translation-filter-suggested-translations": "Voorstelle",
	"cx-suggestionlist-favorite": "Vir later",
	"cx-suggestionlist-title": "Meer voorstelle",
	"cx-suggestionlist-empty-title": "Daar is tans geen voorstelle nie",
	"cx-suggestionlist-empty-desc": "Kies gerus enige onderwerp van u keuse om 'n nuwe vertaling te begin",
	"cx-suggestionlist-empty-desc-recommend-link-text": "Sien voorstelle (eksperimenteel)",
	"cx-suggestionlist-featured": "Uitgelig",
	"cx-suggestionlist-view-source-page": "Sien bronkode",
	"cx-translation-filter-from-any-language": "Vanaf enige taal",
	"cx-translation-filter-to-any-language": "Na enige taal",
	"cx-create-new-translation": "Nuwe vertaling",
	"cx-save-draft-save-success": "{{PLURAL:$1|'n minuut gelede|$1 minute gelede|0=so pas}} Gestoor",
	"cx-save-draft-saving": "Besig om te stoor…",
	"cx-save-draft-tooltip": "Onvoltooide vertalings word outomaties gestoor",
	"cx-save-draft-error": "'n Fout het voorgekom toe die bladsy gestoor is.",
	"cx-contributions-new-contributions": "Nuwe bydrae",
	"cx-contributions-new-article": "Nuwe artikel",
	"cx-contributions-translation": "Vertaling",
	"cx-contributions-upload": "Laai media op",
	"cx-contributions-new-article-tooltip": "Begin om 'n nuwe artikel te skryf",
	"cx-contributions-translation-tooltip": "Vertaal bestaande artikels",
	"cx-contributions-upload-tooltip": "Laai prente, klank- en videolêers op vir gebruik in artikels",
	"cx-publishing-dialog-message": "Die artikel $1 bestaan reeds. Die bestaande inhoud sal met jou vertaling vervang word. Wil jy dit in elk geval publiseer?",
	"cx-publishing-dialog-keep-button": "Hou albei weergawes",
	"cx-publishing-dialog-publish-anyway-button": "Publiseer in elk geval",
	"cx-discard-translation": "Doen weg met vertaling",
	"cx-translation-status-draft": "Aan die gang",
	"cx-translation-status-published": "Gepubliseer",
	"cx-translation-status-deleted": "Geskrap",
	"cx-translation-already-in-progress": "Hierdie artikel word tans vertaal deur $1.",
	"cx-translation-already-in-progress-collaborate": "Koördineer asb. met die gebruiker wat die huidige vertaling gedoen het.",
	"cx-publishing-dialog-publish-draft-button": "Publiseer as konsepartikel",
	"cx-feedback-link": "Gee terugvoer",
	"cx-stats-published-translations-title": "Gepubliseerde vertalings",
	"cx-stats-draft-translations-title": "Vertalings in wording",
	"cx-stats-draft-translations-label": "Vertalings in wording",
	"cx-stats-new-draft-translations-label": "Nuwe vertalings in wording",
	"cx-stats-published-translators-title": "Aantal vertalers",
	"cx-stats-cumulative-tab-title": "Totaal",
	"cx-stats-weekly-trend-tab-title": "Per week",
	"cx-draft-discard-confirmation-message": "Is jy seker dat jy hierdie vertaling permanent wil uitwis?",
	"cx-draft-cancel-button-label": "Kanselleer",
	"cx-draft-discard-button-label": "Wis vertaling uit",
	"cx-beta-feature-enabled-notification": "Inhoudsvertaling is in hierdie wiki geaktiveer.",
	"cx-campaign-newarticle-notice": "Dit is nou makliker om bladsye deur <strong>vertaling</strong> te skep! Lus om die beta-weergawe van die Inhoudvertaler te probeer?",
	"cx-campaign-no-thanks": "Nee dankie",
	"cx-campaign-try": "Probeer Inhoudsvertaling uit",
	"cx-campaign-contributionsmenu-mycontributions": "Bydraes",
	"cx-campaign-contributionsmenu-mytranslations": "Vertalings",
	"cx-campaign-contributionsmenu-translate-instead": "Vertaal eerder as om van voor af te skryf",
	"cx-campaign-contributionsmenu-might-be-available": "$1 is dalk reeds beskikbaar in ander tale",
	"cx-campaign-contributionsmenu-myuploads": "Opgelaaide media",
	"cx-trend-deletions": "Geskrapte vertalings",
	"cx-trend-translations-to": "Vertalings na $1",
	"cx-stats-try-contenttranslation": "Inhoudvertaling is gereedskap om nuwe bladsye te skep deur uit ander tale te vertaal. [$1 Probeer Inhoudvertaling]",
	"cx-stats-all-translations-title": "Vertalings na alle tale",
	"cx-stats-published-translations-label": "Gepubliseerde vertalings",
	"cx-stats-published-target-source": "Vertalings na",
	"cx-stats-published-source-target": "Vertalings vanuit",
	"cx-stats-draft-target-source": "Vertalings in wording na",
	"cx-stats-draft-source-target": "Vertalings in wording vanuit",
	"cx-stats-tabs-expand": "Sien alle tale",
	"cx-stats-total-published": "Totaal gepubliseer",
	"cx-stats-weekly-published": "Laas week",
	"cx-stats-local-published-number": "$1 in $2",
	"cx-stats-local-published": "$1 ($3) in $2",
	"cx-stats-grouping-title": "{{PLURAL:$1|$1 vertaling|$1 vertalings}}",
	"cx-tools-missing-link-tooltip": "Vertaal (in nuwe venster)",
	"cx-tools-missing-link-title": "Ontbrekende skakel",
	"cx-tools-missing-link-mark-link": "{{GENDER:|Merk}} as ontbrekend",
	"echo-category-title-cx": "Vertalings",
	"cx-notification-first-translation": "Geluk met die eerste vertaling! Kyk gerus onder Bydraes en kry nog bladsye om te vertaal.",
	"cx-notification-tenth-translation": "Geluk met die tiende vertaling! Hier is 'n kenner in wording. Hou so aan!",
	"cx-notification-hundredth-translation": "Dit was pas 100 vertalings! Dit is merkwaardig. Deel die nuus met ander medewerkers.",
	"cx-contributions-link": "U bydraes",
	"cx-your-translations-link": "{{GENDER:$1|U bydraes}}",
	"cx-translationlist-empty-title": "Niks om te vertaal nie?",
	"cx-translationlist-empty-desc": "{{GENDER:|Begin}} gerus 'n vertaling nou en werk enige tyd verder daaraan.",
	"cx-tools-link-internal-link": "Skakel na bladsy",
	"cx-tools-link-internal-link-placeholder": "Vind bladsy na die skakel",
	"cx-tools-link-external-link": "Eksterne",
	"cx-tools-link-external-link-placeholder": "Voeg eksterne skakel by.",
	"cx-tools-link-to-another-page": "Skakel na 'n ander bladsy.",
	"cx-tools-link-add-as-missing": "Voeg in as ontbrekende skakel",
	"cx-tools-link-apply": "Gedoen",
	"mw-pageselector-missing": "Bladsy bestaan nie",
	"cx-draft-restoring": "Laai tans die gestoorde vertaling...",
	"cx-draft-restored": "Gestoorde vertaling is gelaai.",
	"cx-draft-restore-failed": "Kon nie die gestoorde vertaling laai nie!",
	"cx-suggestionlist-expand": "Wys almal",
	"cx-suggestionlist-collapse": "Wys minder",
	"cx-suggestionlist-refresh": "Verfris voorstelle",
	"cx-notification-suggestions-available": "Gebaseer op {{GENDER:$2|u}} vorige vertaling van '''$1''', is nuwe voorstelle nou beskikbaar.",
	"cx-tools-linter-title": "Inhoudelike probleme: $1",
	"cx-tools-linter-view-details": "Sien detail",
	"cx-tools-linter-hide-details": "Versteek detail",
	"cx-translator-month-stats-label": "Hierdie maand",
	"cx-translator-total-translations-label": "Totaal",
	"cx-page-old-revision-loaded": "Hierdie vertaling is gebaseer op 'n ouer weergawe van die inhoud. Die bronbladsy het moontlik [$1 aansienlik verander]. U kan voortgaan met dié vertaling, of oorbegin om die bygewerkte inhoud te gebruik.",
	"cx-tools-mt-new-provider": "nuut",
	"cx-tools-mt-new-providers-available": "met nuwe keuses",
	"cx-continue-translation": "Gaan voort met vertaling",
	"cx-tools-template-title": "Sjabloon",
	"cx-template-action-adapt": "Gebruik ekwivalente sjabloon",
	"cx-template-action-keep-original": "Hou oorspronklike sjabloon",
	"cx-template-action-skip": "Slaan sjabloon oor",
	"cx-template-not-available": "Nie beskikbaar in $1 nie",
	"cx-translation-template-add-param": "+ Vertaal veld",
	"cx-translation-template-add-more": "Voeg meer velde by"
}
