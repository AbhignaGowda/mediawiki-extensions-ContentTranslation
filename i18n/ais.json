{
	"@metadata": {
		"authors": [
			"Bunukwiki",
			"Tokoabibi",
			"Benel"
		]
	},
	"cx": "mibelih kasabelih",
	"cx-dashboard-header": "mibelih",
	"cx-header-translation-center": "pikuwanan nu mibelihay",
	"cx-header-new-translation": "baluhayay a kamu-belih",
	"cx-header-all-translations": "sacahamin a belih nu kamu",
	"cx-source-view-page": "baluhayay a kasabelih",
	"cx-publish-page-success": "kasabelih patiyak i $1",
	"cx-publish-button": "patiyak mibelih",
	"cx-publish-button-publishing": "patiyak henay ayza...",
	"cx-publish-gt-move-page-title": "milimad tina kasabelih",
	"cx-source-loading": "miasip henay ayza $1",
	"cx-entrypoint-dialog-button-create-from-scratch": "muliyaw patizeng",
	"cx-tools-searchbox-text": "mikilim masatekeday a sulit",
	"cx-tools-dictionary-title": "misaheci",
	"cx-tools-link-title": "masasiket",
	"cx-tools-link-remove": "misipu masasiket",
	"cx-tools-mt-title": "lunuk mibelih",
	"cx-tools-mt-use-source": "pisaungay nuyaan a sulit",
	"cx-tools-mt-provider-title": "pisaungay $1",
	"cx-stats-title": "lacul mibelih sausi",
	"cx-tools-reference-title": "miazih tu tatenga’ay  nasulitan",
	"cx-tools-reference-remove": "misipu miazih tu tatenga’ay nasulitan",
	"cx-tools-link-hover-tooltip": "sapecec tu miwawah",
	"cx-sourceselector-dialog-new-translation": "baluhayay a kamu-belih",
	"cx-sourceselector-dialog-button-start-translation": "miteka mibelih",
	"cx-sourceselector-dialog-button-cancel": "palawpes",
	"cx-sourceselector-dialog-source-language-label": "patigamiay:",
	"cx-sourceselector-dialog-target-language-label": "katukuh:",
	"cx-sourceselector-dialog-source-title-placeholder": "mikilim saangangan a kasabelih",
	"cx-sourceselector-dialog-target-title-placeholder": "mibelih satangahan",
	"cx-publish-captcha-title": "zahkezay a munday",
	"cx-dashboard-sidebar-stats": "sausi",
	"cx-translation-filter-all-translations": "sacahamin a belih nu kamu",
	"cx-translation-filter-published-translations": "patiyak tuway",
	"cx-translation-filter-draft-translations": "miteka henay ayza",
	"cx-suggestionlist-favorite": "sawnitu mibelih",
	"cx-suggestionlist-empty-title": "inayi’ patahkal nizateng ayza",
	"cx-suggestionlist-featured": "sazumaay a pili’",
	"cx-suggestionlist-view-source-page": "ciwsace tu saayaway kasabelih",
	"cx-translation-filter-from-any-language": "nay amahicahica tu kamu",
	"cx-translation-filter-to-any-language": "katukuh amahicahica tu kamu",
	"cx-create-new-translation": "baluhayay a kamu-belih",
	"cx-save-draft-saving": "misuped henay ayza...",
	"cx-contributions-new-contributions": "baluhayay a paanin",
	"cx-contributions-new-article": "baluhayay a kasabelih",
	"cx-contributions-translation": "mibelih",
	"cx-contributions-upload": "patapabaw tu myiti",
	"cx-publishing-dialog-keep-button": "hizantu tatusaay abaziyong",
	"cx-publishing-dialog-publish-anyway-button": "uyzatu patiyak",
	"cx-discard-translation": "miales tu mibelih",
	"cx-translation-status-draft": "miteka henay ayza",
	"cx-translation-status-published": "patiyak tuway",
	"cx-translation-status-deleted": "masiputu",
	"cx-translation-label-draft": "mibelih miteka henay ayza",
	"cx-translation-label-published": "napatiyakay a belih nu kamu",
	"cx-publishing-dialog-publish-draft-button": "patiyak tu caay kasiwantanay a sulit",
	"cx-feedback-link": "nipabeli nabalucu’an",
	"cx-stats-published-translations-title": "napatiyakay a belih nu kamu",
	"cx-stats-draft-translations-title": "mibelih miteka henay ayza",
	"cx-stats-published-translators-title": "mibelihay nikayadah",
	"cx-stats-cumulative-tab-title": "pulung",
	"cx-stats-weekly-trend-tab-title": "Paylipay",
	"cx-draft-cancel-button-label": "palawpes",
	"cx-draft-discard-button-label": "misipu tu belih nu kamu",
	"cx-campaign-no-thanks": "hatizatu, kukay",
	"cx-campaign-try": "mitanam lacul mibelih",
	"cx-campaign-contributionsmenu-mytranslations": "mibelih",
	"cx-campaign-contributionsmenu-myuploads": "mapatapabaw tu a myiti",
	"cx-trend-deletions": "masiputu tu belih nu kamu",
	"cx-trend-translations-to": "mibelih ta $1",
	"cx-stats-published-translations-label": "napatiyakay a belih nu kamu",
	"cx-stats-published-target-source": "mibelih ta",
	"cx-stats-published-source-target": "mibelih nay",
	"cx-stats-draft-target-source": "imahini mibelih ayza tayza",
	"cx-stats-draft-source-target": "imahini mibelih ayza nay",
	"cx-stats-tabs-expand": "ciwsace tu sacahamin a kamu",
	"cx-stats-total-published": "patiyak tuway pulung nu sausi",
	"cx-stats-weekly-published": "naayaway cacay a lipay",
	"cx-stats-local-published-number": "$2 a $1",
	"cx-tools-missing-link-title": "mahedaw masasiket",
	"echo-category-title-cx": "mibelih",
	"cx-contributions-link": "paanin nu misu",
	"cx-translationlist-empty-title": "inayi’ ku amibelihay?",
	"cx-tools-link-internal-link": "mikilim kasabelih",
	"cx-tools-link-external-link": "masasiket tu hekal",
	"cx-tools-link-external-link-placeholder": "micunus hekalay masasiket",
	"cx-tools-link-add-as-missing": "micunus tu mahedaway masasiket",
	"cx-tools-link-apply": "malahecitu",
	"mw-pageselector-missing": "kasabelih nayai’ tu",
	"cx-suggestionlist-expand": "ciwsace hamin",
	"cx-suggestionlist-collapse": "mikabu",
	"cx-suggestionlist-refresh": "miliyaw lisimet tu patahkal nizateng",
	"cx-tools-linter-title": "munday nu lacul：$1",
	"cx-tools-linter-view-details": "ciwsace mipulita tu kalunasulitan",
	"cx-tools-linter-hide-details": "midimut pulitaay a  kalunasulitan",
	"cx-translator-month-stats-label": "ayza a bulad",
	"cx-translator-total-translations-label": "pulung",
	"cx-tools-mt-new-provider": "baluhay",
	"cx-tools-mt-new-providers-available": "maladay baluhay a kasacacay",
	"cx-continue-translation": "palalid mibelih",
	"cx-publish-destination-namespace-main": "baluhayay a kasabelih"
}
