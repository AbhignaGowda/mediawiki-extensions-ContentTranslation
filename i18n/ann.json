{
	"@metadata": {
		"authors": [
			"Katelem"
		]
	},
	"cx-sx-section-selector-empty-missing-sections-title": "Kpemun̄ ukpa syi owu mônwenebe otu usem",
	"cx-sx-section-selector-pick-other-translation-button-label": "Gobo ofifi akpọk",
	"cx-sx-section-selector-learn-more-anchor-label": "Kween̄ fo lek",
	"cx-translation-list-all-languages-option-label": "Otutuuk",
	"cx-sx-sentence-selector-done-button-label": "Îso",
	"cx-sx-sentence-selector-apply-translation-button-label": "Rọ so",
	"cx-sx-sentence-selector-skip-translation-button-label": "Saba raka",
	"cx-sx-content-comparator-mapped-section-header-title": "Ìkup me $1"
}
