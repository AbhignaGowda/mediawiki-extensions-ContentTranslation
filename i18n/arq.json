{
	"@metadata": {
		"authors": [
			"Oldstoneage"
		]
	},
	"cx-magnus-tool-link-text": "صيب آرتيكلات مخصوصة فل ويكي تاعك"
}
