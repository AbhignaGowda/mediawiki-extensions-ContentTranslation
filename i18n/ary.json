{
	"@metadata": {
		"authors": [
			"Reda Benkhadra"
		]
	},
	"cx-page-selector-widget-nearby-label": "قريب من هنا"
}
