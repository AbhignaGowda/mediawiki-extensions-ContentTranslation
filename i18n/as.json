{
	"@metadata": {
		"authors": [
			"Bishnu Saikia"
		]
	},
	"cx-translation-filter-all-translations": "সমূহ অনুবাদ"
}
