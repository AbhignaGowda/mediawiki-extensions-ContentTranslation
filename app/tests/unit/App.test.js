import App from "@/App.vue";
import i18n from "vue-banana-i18n";
import messages from "@/../../i18n/en.json";

import { shallowMount, createLocalVue } from "@vue/test-utils";

const localVue = createLocalVue();
localVue.use(i18n, { locale: "en", finalFallback: "en", messages });

describe("App component", () => {
  it("has id attribute", () => {
    const wrapper = shallowMount(App, { localVue });
    expect(wrapper.element.id).toBe("cxdashboard");
  });

  it("contains an H1 element", () => {
    const wrapper = shallowMount(App, { localVue });
    expect(wrapper.contains("h1")).toBe(true);
  });

  it("matches the snapshot", () => {
    const wrapper = shallowMount(App, { localVue });
    expect(wrapper.element).toMatchSnapshot();
  });
});
