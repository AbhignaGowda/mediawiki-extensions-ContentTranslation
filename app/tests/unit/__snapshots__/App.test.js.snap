// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component matches the snapshot 1`] = `
<div
  class="container"
  id="cxdashboard"
>
  <cx-header-stub />
   
  <div
    class="row cx-translation-header"
  >
    <mw-button-stub
      class="col-md-4 col-xs-12 col-lg-3 col-offset-lg-1 mb-4 mt-4"
      icon="M11 9V4H9v5H4v2h5v5h2v-5h5V9z"
      label="New translation"
      large="true"
      progressive="true"
      type="button"
    />
  </div>
</div>
`;
